src/App.jsx

```diff
@@ -14,33 +17,90 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function createInternalArrayOfObjectsWithId(arr) {
```

Comment:
Consider renaming the function `createInternalArrayOfObjectsWithId` to start with a verb, such as `createInternalArrayOfObjectsWithId`, to better follow naming conventions.